statistics/cmsketch.go

```diff
@@ -103,6 +103,9 @@ func newTopNHelper(sample [][]byte, numTop uint32) *topNHelper {
 		if i >= numTop && sorted[i]*3 < sorted[numTop-1]*2 && last != sorted[i] {
 			break
 		}
+		if sorted[i] == 1 {
```

Comment:
Combine this with above if branch?

Reply:
But the upper `if` is already too long...